site/docs/reference/authentication.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 1
+---
+
+# Authenticating Flow
+
+Read, write, and admin capabilities over Flow catalogs and the [collections](../concepts/collections.md) that comprise them
+are granted to Flow users through **roles**.
+
+Roles are granted in terms of **prefixes** within the Flow [namespace](../concepts/README.md#namespace).
+By default, each organization, or **tenant**, has a unique high-level prefix.
```

Comment:
nit: I feel like "tenant" is maybe a term best avoided, but I might be out of the loop on that. My 2 cents is "organization" is better.

uber-nit: ~~high-level~~ top-level